templates/cisco_ios_show_ip_bgp_neighbor.textfsm

```diff
@@ -0,0 +1,36 @@
+Value NEIGHBOR_IP (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
```

Comment:
Can you change some of these values to match for say `cisco_nxos_show_ip_bgp_neighbors`? Just so we can try and keep the values as close as possible independent of OS as much as we can.

Reply:
Certainly, that makes a lot of sense. I should have thought to look at some other templates as an example!

Quick question, do you want me to amend my previous commit with these changes or create a new commit that adds them?

Reply:
Adding another commit is fine. We squash them when we merge.